thirtyinch/src/main/java/net/grandcentrix/thirtyinch/internal/DelegatedTiActivity.java

```diff
@@ -33,6 +35,11 @@
     P getRetainedPresenter();
 
     /**
+     * @return {@link UiThreadExecutor}
+     */
+    Executor getUiThreadExecutor();
```

Comment:
This is the same like the `AutoBinder`. Change it to UiTheadExecutor as return value...